pkg/publish/layout.go

```diff
@@ -25,22 +25,27 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/layout"
 	"github.com/google/go-containerregistry/pkg/v1/types"
 	"github.com/google/ko/pkg/build"
+	specsv1 "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
 type LayoutPublisher struct {
-	p layout.Path
+	p    layout.Path
+	tags []string
 }
 
 // NewLayout returns a new publish.Interface that saves images to an OCI Image Layout.
-func NewLayout(path string) (Interface, error) {
+func NewLayout(path string, tags ...string) (Interface, error) {
 	p, err := layout.FromPath(path)
 	if err != nil {
 		p, err = layout.Write(path, empty.Index)
 		if err != nil {
 			return nil, err
 		}
 	}
-	return &LayoutPublisher{p}, nil
+	if len(tags) == 0 {
+		tags = []string{"latest"}
```

Comment:
I'd drop this so it's possible to use this without adding tags?

Reply:
@dentrax are you still interested in working on this?